src/helpers/TypeHelper.ts

```diff
@@ -52,4 +53,24 @@ export class TypeHelper {
 
         return false;
     }
+
+    public static hasDefault(field: Field): boolean {
+        if (field.default === undefined) {
+            return false;
+        }
+
+        return true;
+    }
+
+    public static getDefault(field: Field): string | number | boolean | any[] | null {
```

Comment:
Have you tested the case when default value is null or false? Not sure if it will work as expected.

Reply:
Yeah, works as expected